### PR TITLE
fix: Auto teleport active 

### LINKED ISSUE #8822: Auto teleport active
one issue i came across is this now. Not sure if related to the deploy, but this does not seem right 

image
image
image

seems like its also a canary issue

the autoteleport should not be active 

my address: 
14JipEfjkyFEe4jRViuqFomLPBz96SEXKGcYVkCcfqbAqAxU
nft: 
https://canary.kodadot.xyz/ahp/gallery/50-533

_Originally posted by @exezbcz in https://github.com/kodadot/nft-gallery/issues/8821#issuecomment-1873911753_

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8822

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI;

### CODE REVIEW

**Comment by exezbcz** [comment]
works for me 

image


thanks for the quick fix sir!

**Comment by hassnian** [comment]
@prury pls test the following scenarios 

- not enough funds
- enough funds in current
- enough funds in richest chain , teleport only required
- enough funds in richest chain,  remaining funds will be slashed , teleport all from richest chain

thanks

File: composables/autoTeleport/useAutoTeleportTransitionDetails.ts [hunk 1]
```diff
@@ -214,7 +217,7 @@ export default function (
 
     return [
       hasFetched.teleportTxFee,
-      actionAutoFees.value || hasFetched.actionTxFees,
+      actionAutoFees.value ? hasFetched.actionTxFees : true,
```
Comment (roiLeo) [inline composables/autoTeleport/useAutoTeleportTransitionDetails.ts:220]: always return `true` by default?

**Review by roiLeo — APPROVED** [review]
lgtm

File: composables/autoTeleport/useAutoTeleportTransitionDetails.ts (hunk 1 above)
Comment (hassnian) [reply composables/autoTeleport/useAutoTeleportTransitionDetails.ts:220]: in this case , for nft creation actionTxFees are disabled , and we want to skip that check all together 

can also add `hasFetched.actionTxFees` to an array dynamically if `actionAutoFees.value` is true 

at the end `[].every(Boolean)` would be the same . unless I'm missing something 

Dont wait for `hasFetched.actionTxFees` in this case

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.57 USD/DOT ~ 3.501 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x45eb5a393c70094b503708daac7471892f22c46aa9ecbfe66066b0119bdc0e12](https://polkadot.subscan.io/extrinsic/0x45eb5a393c70094b503708daac7471892f22c46aa9ecbfe66066b0119bdc0e12)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**